m4_sessions/automation/session_one/seven_stephanie_robbie_david_annastasia.md

```diff
@@ -1,3 +1,62 @@
 ## Group Seven's Session Plan
 
 M4 Student Leaders: Stephanie, Robbie, David, Annastasia
+
+# Turing Career Development: Automation
+
+## Session Outcomes
+
+## Mission Alignment		 
+  		  
+*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
+  		  
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+
+## Pre-Session
+
+
+### Survey Questions
+
+Survey Questions to share with instructional staff by Wednesday of the same week.
+
+1.   What value would you assign to this session (from 1-5)?
+2.   What are you taking away from this session?
+3.   What role, if any, does Turing play in taking responsibility for this issue?
+
+
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
```

Comment:
who will send this out to students? is the expectation that students watch/read every resource listed?